docs/content/about.md

```diff
@@ -292,6 +292,89 @@ mvn clean install -DskipTests
 
 The dependencies are now available in your local `.m2` repository.
 
+## How to Contribute
+
+### Code Contribute
+
+- (1) Left comment under the issue that you want to take
+- (2) Fork Flink CDC project to your GitHub repositories
+
+![fork](/_static/fig/contribute_guidance/fork.png "fork")
+
+- (3) Clone and compile your Flink CDC project
+
+```bash
+git clone https://github.com/your_name/flink-cdc-connectors.git
+cd flink-cdc-connectors
+mvn clean install -DskipTests
+```
+
+- (4) Check to a new branch and start your work
+
+```bash
+git checkout -b my_feature
+-- develop and commit
+```
+
+![check_branch](/_static/fig/contribute_guidance/check_branch.png "check_branch")
+
+- (5) Push your branch to your github
+
+```bash
+git push origin my_feature
+```
+
+- (6) Open a PR to https://github.com/ververica/flink-cdc-connectors
+
+![open_pr](/_static/fig/contribute_guidance/open_pr.png "open_pr")
+
+> Code Formatting
+
+- You will also need to install the google-java-format plugin. However, a specific version of this plugin is required. Download google-java-format v1.7.0.6 and install it as follows. Make sure to never update this plugin.
+
+> Flink cdc uses Spotless together with google-java-format to format the Java code.
+
+It is recommended to automatically format your code by applying the following settings:
+
+- 1.Go to “Settings” → “Other Settings” → “google-java-format Settings”.
+- 2.Tick the checkbox to enable the plugin.
+- 3.Change the code style to “Android Open Source Project (AOSP) style”.
+- 4.Go to “Settings” → “Tools” → “Actions on Save”.
+- 5.Under “Formatting Actions”, select “Optimize imports” and “Reformat file”.
+- 6.From the “All file types list” next to “Reformat code”, select Java.
+
+For earlier IntelliJ IDEA versions: 4. Go to “Settings” → “Other Settings” → “Save Actions”. 5. Under “General”, enable your preferred settings for when to format the code, e.g. “Activate save actions on save”. 6. Under “Formatting Actions”, select “Optimize imports” and “Reformat file”. 7. Under “File Path Inclusions”, add an entry for .*\.java to avoid formatting other file types.
```

Comment:
```
For earlier IntelliJ IDEA versions, the step 4 to 7 will be changed as follows. 

- 4. Go to “Settings” → “Other Settings” → “Save Actions”. 
- 5. Under “General”, enable your preferred settings for when to format the code, e.g. “Activate save actions on save”. 
- 6. Under “Formatting Actions”, select “Optimize imports” and “Reformat file”. 
- 7. Under “File Path Inclusions”, add an entry for .*\.java to avoid formatting other file types.
```